storage/replica.go

```diff
@@ -862,14 +865,22 @@ func (r *Replica) proposeRaftCommand(ctx context.Context, ba roachpb.BatchReques
 	}
 	cmdID := ba.GetOrCreateCmdID(r.store.Clock().PhysicalNow())
 	idKey := makeCmdIDKey(cmdID)
-	r.Lock()
-	r.pendingCmds[idKey] = pendingCmd
-	r.Unlock()
+
 	var errChan <-chan error
-	if r.proposeRaftCommandFn != nil {
-		errChan = r.proposeRaftCommandFn(idKey, raftCmd)
+	if r.quiesced {
```

Comment:
We don't have many, but a lot of the ones in `client_raft_test.go` do. But I guess we don't have any tests that unreplicate while something else is going on.